src/locale/properties/record/record.en.properties

```diff
@@ -86,3 +86,7 @@ register.pleaseReviewInvalidForm=Please review the form and fix the issues befor
 register.invalidForm=Form validation error
 record.unidentifiedOrganization=Unidentified organization
 record.identifyAS=Identify as:
+record.accountLocked1=Your account has been locked
```

Comment:
@ leomendoza123 Could you change the name of the properties to a more descriptive name like the other properties to have them homogenized